spec/models/schedule_generator_service_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe ScheduleGenerator, type: :model do
+  describe '#generate_schedule' do
+    subject(:generate_schedule) { ScheduleGenerator.new(current_member).generate_schedule }
+
+    let(:current_member) { create :member }
+
+    before { allow(DateTime).to receive(:current).and_return DateTime.new(2020, 12, 10) }
+
+    it 'generates missions for the next month' do
+      expect { generate_schedule }.to change(Mission, :count).by(78)
```

Comment:
It might be helpful to have a small explanation on why we expect the created Mission count to be `78`